manifests/rhoai/shared/base/nvidia-nim-validator-cron.yaml

```diff
@@ -33,6 +33,105 @@ spec:
                 RESULT_CONFIGMAP_NAME='nvidia-nim-validation-result'
                 DATA_CONFIGMAP_NAME='nvidia-nim-images-data'
                 IMAGE_PULL_SECRET_NAME='nvidia-nim-image-pull'
+                SERVING_RUNTIME_TEMPLATE_NAME='nvidia-nim-serving-template'
+                SERVING_RUNTIME_TEMPLATE_YAML='
+                  {
+                    "apiVersion": "template.openshift.io/v1",
+                    "kind": "Template",
+                    "metadata": {
+                      "annotations": {
+                        "opendatahub.io/apiProtocol": "REST",
+                        "opendatahub.io/modelServingSupport": "[\"single\"]"
+                      },
+                      "name": "nvidia-nim-serving-template"
+                    },
+                    "objects": [
+                      {
+                        "apiVersion": "serving.kserve.io/v1alpha1",
+                        "kind": "ServingRuntime",
+                        "labels": {
+                          "opendatahub.io/dashboard": "true"
+                        },
```

Comment:
@xieshenzh https://redhat-internal.slack.com/archives/C07CZDA6DE1/p1725994445465669?thread_ts=1725991550.539169&cid=C07CZDA6DE1

Reply:
This label is for ServingRuntime, not for the Template. I removed the label for the Template.
If my understanding is correct, Templates without the label should not be shown in `Settings` -> `Serving Runtimes` list. And I tested it in our lab last week, it worked as expected.

The NIM ServingRuntime are supposed to be created with this label via dashboard, right?